### PR TITLE
Chapter 9

### PR DESCRIPTION
**Pull request by aisharina99**
ready

### CODE REVIEW

**Comment by aisharina99** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-  end
+  def new; end
 
   def create
-    user = User.find_by email: params[:session][:email].downcase
-    if user&.authenticate params[:session][:password]
-      log_in user
-      redirect_to user
+    @user = User.find_by email: params[:session][:email].downcase
+    if @user &.authenticate params[:session][:password]
```
Comment (huongnt-2545) [inline app/controllers/sessions_controller.rb]:
```suggestion
    if @user&.authenticate(params[:session][:password])
```

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+      params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
+      redirect_to @user
     else
       flash.now[:danger] =  "Invalid email/password combination"
       render :new
     end
   end
 
   def destroy
-    log_out if logged_in?
```
Comment (huongnt-2545) [inline app/controllers/sessions_controller.rb]: keep this, don't change

**Comment by aisharina99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
please merge this pull

**Pull request merged**